docs/blog/.authors.yml

```diff
@@ -3,3 +3,7 @@ authors:
     name: Sid
     description: Data Science Director
     avatar: https://raw.githubusercontent.com/IDinsight/ask-a-question/main/docs/blog/author-avatars/sid.png
+  Poornima:
+    name: Poornima
+    description: Data Scientist
+    avatar: ./blog/author-avatars/poornima.png
```

Comment:
So relative links don't work here :(
Known issue with mkdocs

you could use a permalink for where it _will_ end up once this is merged.

e.g. https://raw.githubusercontent.com/IDinsight/experiments-engine/main/docs/blog/author-avatars/poornima.png

Mind also updating mine from the AAQ repo link to the ExE one?